### PR TITLE
Fixed bug concerning moved rack beyond specified boundaries

### PR DESCRIPTION
**Pull request by ar4s**
Connected to #1798

### CODE REVIEW

**Comment by mkurek** [comment]
:ok:

File: src/ralph/dc_view/static/js/app/data_center/directives.js [hunk 1]
```diff
@@ -30,25 +30,32 @@
...
                             var isSelectedRack = value.id !== scope.rack.id;
-                            var isOccupiedSpace = value.visualization_col == position_x && value.visualization_row == position_y;
+                            var isOccupiedSpace = value.visualization_col == x && value.visualization_row == y;
                             if (isSelectedRack && isOccupiedSpace) {
-                                can_drop = false;
+                                return false;
                             }
                         });
-                        if(can_drop) {
+                        if (x >= scope.dc.visualization_cols_num || y >= scope.dc.visualization_rows_num)
```
Comment (ghost) [inline src/ralph/dc_view/static/js/app/data_center/directives.js]: This should be above forEach. But it's only optimisation. In js :D

**Comment by ghost** [comment]
I pointed a minor thing. If you think that comment is worthless, merge by yourself.
:ok:

File: src/ralph/dc_view/static/js/app/data_center/directives.js (hunk 1 above)
Comment (ar4s) [inline src/ralph/dc_view/static/js/app/data_center/directives.js]: Changed

**Pull request merged**